src/mngt/ocf_mngt_core.c

```diff
@@ -541,8 +541,8 @@ void ocf_mngt_cache_add_core(ocf_cache_t cache,
 		goto err_pipeline;
 	}
 
-	result = env_strncpy(name, OCF_CORE_NAME_SIZE,
-			cfg->name, OCF_CORE_NAME_SIZE);
+	result = env_strncpy(name, cfg->name_len,
```

Comment:
second param should be OCF_CORE_NAME_SIZE

Reply:
done

Reply:
I meant second out of four, not the second one that you change :)

env_strncpy(name, OCF_CORE_NAME_SIZE, cfg->name, cfg->name_len)

Reply:
my mistake, now fixed